tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('prepare-deploy:devapp', ['build:devapp'], () => {
```

Comment:
@DevVersion we can still have the gulp task for just the build part- maybe `stage:devapp`? I just suspect people wouldn't need to use it much